src/shaders/gfx_batch/hbao/bilateralblur.frag

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
Just so we don't forget: run all these through

```sh
magnum-shaderconverter --validate --input-version "300 es" --output-version opengl <shader-file>
```

and fix the issues it complains about, so the initial version is without WebGL shader compiler errors. May also need to add various `-D` definitions depending on the shader, such as `-DAO_LAYERED=1 -DAO_INTERLEAVED=1` etc. for `hbao.frag`.

Reply:
This has been completed, and with the exception of the geometry shader, they've all been made compatible.